app/models/nonprofit.rb

```diff
@@ -82,7 +82,7 @@ def build_admin(**kwargs)
   end
   has_many :tag_masters, dependent: :destroy
   has_many :custom_field_masters, dependent: :destroy
-  
```

Comment:
Why all these formatting changes?

Reply:
I'm sorry about that, I didn't mean to change it. I think somehow I've activated an auto formatter. I'll fix this.

Reply:
About the test, I think it's a great idea! We'll be adding it.

Reply:
Sorry for the delay, we were kinda busy these days.

Reply:
No need to apologize! Your contributions are super valuable and we're thankful you're willing to volunteer and help out!